go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/cilium/cilium
 
 // renovate: datasource=golang-version depName=go
-go 1.21.0
+go 1.21.3
```

Comment:
The patch version should be always set to 0 AFAIR, to avoid forcing every developer to align with that exact patch version.

Reply:
I think this is lower bound by the version from all go.mods of the dependencies, hence from https://github.com/cilium/stream/blob/main/go.mod in this case. Would need to change that, first.

Reply:
:heavy_plus_sign: Please set this to 0.

Reply:
Thanks for catching this! I've fixed this in cilium/stream, bumped deps and fixed the version here.

Reply:
Thanks!